pkg/names/names.go

```diff
@@ -68,6 +68,7 @@ var (
 		// Prevent "MultipartUpload" from becoming "MultIPartUpload"
 		{"Ip", "IP", "ip", regexp.MustCompile("Ip(?!art)", regexp.None)},
 		// Easy find-and-replacements...
+		{"Ami", "AMI", "ami", nil},
```

Comment:
@brycahta @RedbackThomson I think this is going to have issues with "Family"... which this is going to turn into "FAMIly"

Reply:
We can use something similar to what we used for URI/SecURIty: `{"Uri", "URI", "uri", regexp.MustCompile("(?!sec)uri(?!ty)|(Uri)", regexp.None)},`

Reply:
Good points, I was surprised no tests were failing tbh. Let me submit a revision to handle "Family"

Reply:
https://github.com/aws-controllers-k8s/code-generator/pull/167